src/elements/content-uploader/UploadsManagerAction.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @flow
+ * @file Uploads Manager action component
+ */
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import messages from '../common/messages';
+import PrimaryButton from '../../components/primary-button';
+import { STATUS_ERROR } from '../../constants';
+
+type Props = {
+    numFailedUploads: number,
+    onClick: Function,
+};
+
+const UploadsManagerAction = ({ onClick, numFailedUploads }: Props) => {
+    const handleResumeClick = event => {
+        event.stopPropagation();
+        onClick(STATUS_ERROR);
+    };
+
+    const resumeMessage =
```

Comment:
Similar to a prior comment:

const resumeMessage = numFailedUploads > 1 ? message.resumeAll : messages.resume;
...
<FormattedMessage {...resumeMessage} />

Reply:
And additionally, does numFailedUploads need to be passed in as-is, or can we have a boolean prop `hasMultipleUploadsFailed` which abstracts out how its calculated from the perspective of this function